bpf/Makefile

```diff
@@ -58,7 +59,7 @@ LB_OPTIONS = \
 # tested as well.
 MAX_LB_OPTIONS = -DENABLE_IPV4 -DENABLE_IPV6 -DLB_L3 -DLB_L4 \
 	-DENABLE_HOST_SERVICES_TCP -DENABLE_HOST_SERVICES_UDP \
-	-DENABLE_IPSEC
+	-DENABLE_IPSEC -DIPV4_FRAGMENTS
```

Comment:
You can't enable IPv4 fragments here because this set of options is tested on 4.9. This is likely the reason why the `RuntimeVerifier` test currently fails.

Related: #10798.

Reply:
Oh I missed that. Ok, will remove until that issue is fixed. Do you know if leaving the directive in `LB_OPTIONS` and `LXC_OPTIONS` is safe?

Reply:
Yes, it should be safe. Only the `MAX_*_OPTIONS` are used for the `RuntimeVerifier` test.